### DISCUSSION TITLE
Problema  Trojan.Agent.Win32.3991781

### DISCUSSION THREAD

**Discussion by manuelbernalcarvajal** [theJayTea/WritingTools#31] [General]
Bueno, he venido a probar este proyecto que me parece muy interesante, y que aplaudo el esfuerzo, sin embargo me parece adecuado decirles que al analizar el software de la version v4 a traves de VirusTotal, como ejemplo, da positivo por según tal programa 
 un Trojan.Agent.Win32.3991781, considero que es correcto hacerselo saber, y por ello les dejo este mensaje.

Muchas gracias por todo, ahora lo pongo en una **version en ingles.**

Well, I have come to try this project that I find very interesting, and I applaud the effort, however I think it is appropriate to tell you that when analyzing the software of version v4 through VirusTotal, as an example, it gives positive for according to that program
a Trojan.Agent.Win32.3991781, I consider that it is correct to let you know, and for that reason I leave you this message.

Thank you very much for everything, above is the original **version in Spanish**.
![image](https://github.com/user-attachments/assets/bfe18ae7-e706-4db3-8e12-0222f492edee)

**Comment by theJayTea** [comment]
Hi! Thanks for sharing this. That’s a false positive due to ML heuristics.
Only 1 out of 64 AVs has anything to say. The one that says something is an obscure AV, and its description has “static ML” in it, which means it’s using an ML heuristics model, which is known for a lot of false positives.

**Reply by manuelbernalcarvajal** [reply]
Thank you very much for answering, I feel more at ease.
